lib/src/config.dart

```diff
@@ -176,20 +200,23 @@ void _parseDeletionPatterns(yaml, GlobalConfig config) {
       .addAll((patterns as YamlList).map((pattern) => new RegExp(pattern)));
 }
 
-void _parseFilesToLoad(yaml, GlobalConfig config) {
+Future _parseFilesToLoad(yaml, GlobalConfig config) async {
   var filePaths = _parseStringList(yaml, 'files_to_load');
   if (filePaths == null) return;
-  config.filesToLoad.addAll(filePaths.map((filePath) {
+  config.filesToLoad.addAll(await Future.wait(filePaths.map((filePath) async {
     var parts = filePath.split(':');
     if (parts.length == 1) {
       return platformIndependentPath(parts[0]);
     } else if (parts.length == 2 && parts[0] == 'package') {
-      return path.join('packages', platformIndependentPath(parts[1]));
+      Uri uri = await config.packageResolver.resolveUri(filePath);
+      String p = platformIndependentPath(uri.toFilePath());
+      p = p.replaceAll("%", r'%25'); // Reset url encoding
```

Comment:
My question is why does the url encoding need to be reset? I am curious what the uri looks like before/after this line. I wouldn't expect any `%` in there at all intuitively. I haven't used the packageResolver package before but it seems like its giving a weird path here, and this is some sort of workaround?